### ISSUE TITLE
Code-fenced blocks are counted as paragraphs

### ISSUE THREAD

**Issue by Michionlion** [gkapfham/gatorgrader#17]
When doing sentence or paragraph checking, code fenced blocks would be counted as paragraphs, and therefore be checked for the requisite number of sentences. This should not happen. An example code block that would fail a check for 3 sentences despite every paragraph being 3 sentences is shown below.

````
# Header, should not be considered a paragraph

This is a paragraph. It will be counted as a paragraph.
Then it will also be checked to ensure it has a certain number of sentences (in this case, 3).

```
This is a code block - it should not be considered a paragraph, but it is, and therefore also fails the sentence check.
```

````

**Comment by Michionlion** [comment]
I've create a new [branch](/gkapfham/gatorgrader/tree/i-17-code-fence-bug) to fix this bug.

**Comment by Michionlion** [comment]
This bug has been fixed in PR #21. Therefore, I am closing the issue.
